### PR TITLE
Kubernetes Runner

### PR DESCRIPTION
**Pull request by edeediong**

### CODE REVIEW

File: cli/test/test_runner_kubernetes.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+class TestKubernetesDockerRunner(unittest.TestCase):
+    def setUp(self):
+        log.setLevel('CRITICAL')
+
+    @unittest.skipIf(
+        os.environ['ENGINE'] != 'docker' or os.environ['RESMAN'] != 'kubernetes',
+        'ENGINE != docker or RESMAN != kubernetes')
+    def test_docker_basic_run(self):
+        repo = testutils.mk_repo()
+        conf = PopperConfig(workspace_dir=repo.working_dir)
```
Comment (ivotron) [inline cli/test/test_runner_kubernetes.py:22]: thanks a lot again @edeediong for working on this, really appreciate it! 

In order to instruct popper to use the kubernetes runner, we can need to add `resman_name='kubernetes'` to these arguments.

Comment (edeediong) [reply cli/test/test_runner_kubernetes.py:22]: Alright sure!

File: cli/test/test_runner_kubernetes.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import os
+import unittest
+
+import utils as testutils
```
Comment (ivotron) [inline cli/test/test_runner_kubernetes.py:4]: also, I forgot to mention, this module disappeared in one of our recent changes to `master`

File: cli/test/test_runner_kubernetes.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+from popper.config import PopperConfig
+from popper.parser import YMLWorkflow
+from popper.runner import WorkflowRunner
+
+from popper.cli import log as log
+
+
+class TestKubernetesDockerRunner(unittest.TestCase):
```
Comment (ivotron) [inline cli/test/test_runner_kubernetes.py:13]: there is a `PopperTest` that we can derive from that has the `mk_repo` function as a member. Take a look at how this is done [here](https://github.com/systemslab/popper/blob/master/cli/test/test_runner_host.py#L19).

File: cli/test/test_runner_kubernetes.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    def setUp(self):
+        log.setLevel('CRITICAL')
+
+    @unittest.skipIf(
+        os.environ['ENGINE'] != 'docker' or os.environ['RESMAN'] != 'kubernetes',
+        'ENGINE != docker or RESMAN != kubernetes')
+    def test_docker_basic_run(self):
+        repo = testutils.mk_repo()
```
Comment (ivotron) [inline cli/test/test_runner_kubernetes.py:21]: once we derive from `PopperTest`, then this would become `self.mk_repo()`

Comment (edeediong) [reply cli/test/test_runner_kubernetes.py:21]: Done

File: cli/test/test_runner_kubernetes.py (hunk 3 above)
Comment (edeediong) [reply cli/test/test_runner_kubernetes.py:13]: Done

File: cli/test/test_runner_kubernetes.py (hunk 2 above)
Comment (edeediong) [reply cli/test/test_runner_kubernetes.py:4]: Noted

File: cli/test/test_runner_kubernetes.py (hunk 1 above)
Comment (edeediong) [reply cli/test/test_runner_kubernetes.py:22]: done

Comment (edeediong) [reply cli/test/test_runner_kubernetes.py]: If I add `resman_name='kubernetes'` to `PopperConfig` as a parameter, it sorta breaks the codebase.

**Pull request closed without merging**